### PR TITLE
[doc] consistent True/False/None default format

### LINKED ISSUE #14949: [doc] post-conversion: incosistent "default to"
After the conversion we ended up with inconsistent values for `defaults to` - sometimes it's `formatted`, other times it's *italics*, with former being the prevailing form. Example:
```
src/transformers/generation_tf_utils.py:            use_cache: (`bool`, *optional*, defaults to `True`):
src/transformers/generation_tf_utils.py:            output_attentions (`bool`, *optional*, defaults to *False*):
```
but there is a lot more of these

@sgugger

### PR DESCRIPTION
**Pull request by stas00**
This PR sets True/False/None default format in a consistent manner of backticks.

```
find . -type d -name ".git" -prune -o -type f -exec perl -pi -e 's#\*(True|False|None)\*#`$1`#g' {} \;
```

Fixes: https://github.com/huggingface/transformers/issues/14949

@sgugger

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Output looks good. Could you run the command again on a fresh master? I'll check again the output :-)

**Comment by stas00** [comment]
Done

File: src/transformers/models/xlnet/modeling_xlnet.py [hunk 1]
```diff
@@ -1383,7 +1383,7 @@ def forward(
         r"""
         labels (`torch.LongTensor` of shape `(batch_size, num_predict)`, *optional*):
             Labels for masked language modeling. `num_predict` corresponds to `target_mapping.shape[1]`. If
-            `target_mapping` is :obj*None*, then `num_predict` corresponds to `sequence_length`.
+            `target_mapping` is :obj`None`, then `num_predict` corresponds to `sequence_length`.
```
Comment (sgugger) [inline src/transformers/models/xlnet/modeling_xlnet.py:1386]:
```suggestion
            `target_mapping` is `None`, then `num_predict` corresponds to `sequence_length`.
```

**Review by sgugger — APPROVED** [review]
Thanks a lot!

**Pull request merged**